packages/server/api/src/app/flows/flow/flow.service.ts

```diff
@@ -660,6 +662,7 @@ type CreateParams = {
   userId: UserId;
   projectId: ProjectId;
   request: CreateEmptyFlowRequest;
+  isWorkflow?: boolean;
```

Comment:
made it optional, realistically we dont wanna change all the calls to this, so only set it when you want it to be false